test/system/200-pod.bats

```diff
@@ -6,13 +6,7 @@ load helpers
 function teardown() {
     run_podman pod rm -f -t 0 -a
     run_podman rm -f -t 0 -a
-    run_podman image list --format '{{.ID}} {{.Repository}}'
-    while read id name; do
-        if [[ "$name" =~ /podman-pause ]]; then
-            run_podman rmi $id
-        fi
-    done <<<"$output"
-
+    run_podman rmi $(pause_image)
```

Comment:
One more concern: this `rmi` will fail if `pause_image` does not exist, e.g., if some test fails in a step before creating a pod. Unlikely, but if it does happen, `teardown` will fail, and that can make a mess. With a 'grep images' approach, a failure is less likely.

I'm not sure what to recommend here. One approach would be to add `run_podman rmi $(pause_image)` to each subtest. I like that, because it moves cleanup into each test, but it's probably more than you signed up for in this PR. 

I tentatively suggest just forgetting about this part of the PR, and letting me file a new PR later that will handle cleanup. I need to do that anyway, because we have some ugly-read-yuk right now because one of these tests uses a `k8s.something/pause` which this teardown does not clean up.

Reply:
Great catch!  We already have an `--ignore` flag for `container rm` and this screams like we should have one for `image rm` as well.

Reply:
`run_podman ? rmi` could work